tensorrt_llm/_torch/attention_backend/sparse/dsa.py

```
@@ -528,24 +528,21 @@ def create_expanded_buffers(self, capture_graph=False):
-        spec_decoding_tensor: Optional['SpecDecodingTensor'] = None,
-    ):
+            self,
+            batch_size,
+            is_spec_decoding_enabled,
+            is_spec_dec_tree,
+            is_spec_dec_dynamic_tree,
+            max_draft_len,
+            max_total_draft_tokens,
+            is_target_model: bool = True,
+            model_is_wrapped: bool = False,
+            spec_tree_manager: Optional['SpecTreeManager'] = None):
```

Ruff (0.15.5)

[error] Undefined name `SpecTreeManager`

(F821)